next/pages/channel.js

```diff
@@ -41,6 +46,10 @@ const StyledAuthor = styled(Label)`
   margin: 0;
 `
 
+const StyledProfileThumb = styled(Image)`
+  border-radius: 50% 50%;
```

Comment:
Is there a reason to use the value twice here?

Reply:
Nope, no reason. Probably I repeated intentionally.